docs/guide/README.md

```diff
@@ -2,8 +2,7 @@
 
 A guide introducing basic concepts for working with Contentful.
 
-This command is intended for users new to Contentful, regular users can run
-`contentful boilerplate` directly to quickly set up an example project.
+This command is intended for users new to Contentful.
```

Comment:
why would we keep this line?

Reply:
I wasn't sure about it either but now when reading it a second time I think we should remove this line completely. Thank you